### PR TITLE
Add azure teardown method to ensure running status of all VM instances

### PR DESCRIPTION
**Pull request by suchita-g**
Signed-off-by: Suchita Gatfane <sgatfane@redhat.com>

### CODE REVIEW

**Review by ebenahar — APPROVED** [review]

**Review by jilju — DISMISSED** [review]

**Review by jilju — DISMISSED** [review]

**Review by jilju — APPROVED** [review]

**Review by wusui — APPROVED** [review]
Looks good

File: ocs_ci/ocs/platform_nodes.py [hunk 1]
```diff
@@ -1907,6 +1907,29 @@ def attach_nodes_to_cluster(self, node_list):
             "attach nodes to cluster functionality is not implemented"
         )
 
+    def restart_nodes_by_stop_and_start_teardown(self):
+        """
+        Make sure all VM instances up by the end of the test
+
+        """
+        self.cluster_nodes = get_node_objs()
+        vms = self.get_vm_names()
```
Comment (sidhant-agrawal) [inline ocs_ci/ocs/platform_nodes.py:1916]: Shouldn't this be  `vms = self.azure.get_vm_names()` ?

**Review by ebenahar — APPROVED** [review]

File: ocs_ci/ocs/platform_nodes.py (hunk 1 above)
Comment (suchita-g) [reply ocs_ci/ocs/platform_nodes.py]: Good catch. updated.

**Review by sidhant-agrawal — APPROVED** [review]

**Review by jilju — APPROVED** [review]

**Comment by suchita-g** [comment]
PR Validation failed, but the test was passed.
 Verified:  https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/16755/

**Review by sidhant-agrawal — APPROVED** [review]

**Review by jilju — APPROVED** [review]

**Review by ebenahar — APPROVED** [review]

**Pull request merged**